### ISSUE TITLE
Valid placeholder extend  flagged as incorrect

### ISSUE THREAD

**Issue by danielguillan** [brigade/scss-lint#178]
The following code is flagged as incorrect but we are using a _valid_ placeholder selector with `@extend`:

``` scss
$a: 'some';
$b: 'placeholder';
$dynamically-generated-placeholder-name: '%' + $a + '-' + $b;

#{$dynamically-generated-placeholder-name} {
  color: #fff;
}

.test {
  @extend #{$dynamically-generated-placeholder-name};
}
```

It might be impossible for Scss-lint to guess if the interpolated variable is in fact a placeholder or not but I think it shouldn't flag it as invalid.
